ykcompile/src/lib.rs

```diff
@@ -78,6 +79,8 @@ pub struct TraceCompiler {
     available_regs: Vec<u8>,
     /// Maps locals to their assigned registers.
     assigned_regs: HashMap<Local, u8>,
+    /// Stores return destinations from calls
```

Comment:
full stop.

Reply:
I think I get why this is needed (so we know where to copy rax to), but it'd help to expand the comment.